packages/@react-spectrum/menu/docs/MenuTrigger.mdx

```diff
@@ -121,6 +121,63 @@ import {Text} from '@react-spectrum/text';
 </MenuTrigger>
 ```
 
+## Unavilalbe Menu Item <VersionBadge version="alpha" style={{marginLeft: 4, verticalAlign: 'bottom'}} />
+
+ContextualHelpTrigger shows a user extra information about the state of an unavilalbe Menu Item.
```

Comment:
I would probably include something about why you would want to use this (when you want to keep an item in the UI, but it isn't available to a user + you have the ability to add context as to why it's unavailable). This currently sounds like there is another way to make items unavailable, but this is how you add extra information.